docs/quick-start.md

```diff
@@ -11,7 +11,10 @@ gRPC requires Swift 5.0 or higher.
 #### Install Protocol Buffers v3
 
 Install the protoc compiler that is used to generate gRPC service code. The
-simplest way to do this is to download pre-compiled binaries for your
+simplest way to do this is to use Homebrew.
```

Comment:
This is only the simplest way if you're using macOS (Homebrew exists on Linux too but I'm not sure it's so ubiquitous) and have Homebrew installed so I think we should prefix this by saying something to that effect.